src/main/java/com/atlassian/db/replica/internal/aurora/AuroraEndpoints.java

```diff
@@ -0,0 +1,20 @@
+package com.atlassian.db.replica.internal.aurora;
+
+import static com.atlassian.db.replica.internal.aurora.AuroraCluster.AuroraClusterBuilder.anAuroraCluster;
+import static com.atlassian.db.replica.internal.aurora.AuroraEndpoint.AuroraEndpointBuilder.anAuroraEndpoint;
+
+public class AuroraEndpoints {
+    private AuroraEndpoints() {
+    }
+
+    /**
+     * Transforms reader endpoint to instance endpoint
+     */
+    public static AuroraEndpoint instanceEndpoint(String readerEndpoint, String serverId) {
```

Comment:
So this procedural util function is equivalent to static `AuroraEndpoint.parse(String readerEndpoint, String serverId)`.